### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by jackyykk**
Update the game over message so people know how to play again!
Click Create pull request.

### CODE REVIEW

**Comment by motocat** [comment]
tested and it seems safe to merge

**Review by jackyykk — COMMENTED** [review]
it seems that it's safe to merge

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (motocat) [inline index.html:51]: please add more feelings

**Review by motocat — CHANGES_REQUESTED** [review]
Please add feelings

**Review by motocat — APPROVED** [review]
Approved

**Pull request merged**